src/HtmlGenerator.Tests/GitBlameVisitorTests.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using GitGlyph;
+using LibGit2Sharp;
+using Microsoft.SourceBrowser.HtmlGenerator.Utilities;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace HtmlGenerator.Tests
+{
+    [TestClass]
+    public class GitBlameVistorTests
+    {
+        [TestMethod]
+        public void TestVisit()
+        {
+            string pathToTempDirectory = Path.Combine(Path.GetTempPath(),"GitBlameVisitorRepo\\");
+            string pathToTestFile = Path.Combine(pathToTempDirectory,"Test.cs");
+            string pathToGitBlameRepo = Directory.GetCurrentDirectory() + "\\Repositories\\GitBlameVisitorRepo.git";
+            try
+            {
+                Repository.Clone(pathToGitBlameRepo, pathToTempDirectory);
+                Repository repo = new Repository(Path.Combine(pathToTempDirectory,".git"));
+                //Test.cs contains:
+                //bd46cc8 public class Test
+                //bd46cc8 {
+                //909a6197  private int test;
+                //11933d2a  private int myInt;
+                //bd46cc8 }
+                Dictionary<string, string> context = new Dictionary<string, string>
+                {
+                    {Microsoft.SourceBrowser.MEF.ContextKeys.FilePath, pathToTestFile},
+                    {Microsoft.SourceBrowser.MEF.ContextKeys.LineNumber, "3"}
+                };
+                string expectedThirdLineSha1 = "909a6197";
+                GitBlameVisitor visitor = new GitBlameVisitor(repo, new PluginLogger());
+                string htmlResult = visitor.Visit(null, context);

```

Comment:
OPT: you can put back the spaces around line 37 to separate the 3 steps: setup, test, assert